src/sentry/workflow_engine/models/action.py

```diff
@@ -45,3 +54,12 @@ class Type(models.TextChoices):
 
     # LEGACY: This is used to denote if the Notification is going to a user, team, sentry app, etc
     target_type = models.SmallIntegerField(choices=ActionTarget.as_choices(), null=True)
+
+    def get_handler(self) -> ActionHandler:
+        action_type = Action.Type(self.type)
+        return action_handler_registry.get(action_type)
```

Comment:
should this catch `NoRegistrationExistsError` if the handler doesn't exist? maybe we can enforce registries having handlers in tests?

Reply:
I had purposefully let this fall through, since we control the Action.Type information on save, it would only be invalid if something unexpected / not good is happening. I was thinking, have this fail loudly so we know to investigate / fix.